src/libcollections/btree/node.rs

```diff
@@ -608,7 +543,8 @@ impl<'a, K, V> NodeRef<marker::Mut<'a>, K, V, marker::Internal> {
         unsafe {
             ptr::write(self.keys_mut().get_unchecked_mut(idx), key);
             ptr::write(self.vals_mut().get_unchecked_mut(idx), val);
-            ptr::write(self.as_internal_mut().edges.get_unchecked_mut(idx + 1), edge.node);
+            ptr::write(self.as_internal_mut().edges.get_unchecked_mut(idx + 1),
+                       edge.node);
```

Comment:
I don't think that this is a good change, if only for consistency with the above lines.